src/SourcesCommand.cc

```diff
@@ -50,7 +51,12 @@ class SourcesCommand : public Command {
   static SourcesCommand singleton;
 };
 
-SourcesCommand SourcesCommand::singleton("sources", " rr sources [<trace_dir>]\n");
+SourcesCommand SourcesCommand::singleton(
+    "sources",
+    " rr sources [<trace_dir>]\n"
+    "  --substitute=LIBRARY=PATH  When searching for the source to LIBRARY,\n"
+    "                             substitute PATH in place of the path stored\n"
+    "                             in the library's DWARF debug information.\n");
```

Comment:
Be specific here and say that this replaces `DW_AT_comp_dir`.

Reply:
... for all compilation units in that binary.

Reply:
done.